src/coreclr/src/vm/method.cpp

```diff
@@ -5254,6 +5254,7 @@ void NDirectMethodDesc::InterlockedSetNDirectFlags(WORD wFlags)
 }
 
 #ifndef CROSSGEN_COMPILE
+#ifndef TARGET_UNIX
```

Comment:
Nit: I would use `#ifdef TARGET_WINDOWS` for these. I think it reads better to use Windows ifdef for Window-specific stuff.